apps/www/content/docs/components/sheet.mdx

```diff
@@ -84,7 +84,7 @@ You can adjust the size of the sheet using CSS classes:
 ```tsx {3}
 <Sheet>
   <SheetTrigger>Open</SheetTrigger>
-  <SheetContent className="w-[400px] sm:w-[540px]">
+  <SheetContent className="w-[400px] sm:w-[540px] sm:max-w-none">
```

Comment:
I'm wondering if the sm:max-w-none class is still necessary here. It seems like the sm:w-[540px] class is already setting the maximum width of the component to 540px, so the sm:max-w-none class may not be needed.
```suggestion
  <SheetContent className="w-[400px] sm:w-[540px]">
```
BTW great catch 🚀